lib/secure_headers/headers/policy_management.rb

```diff
@@ -199,9 +153,9 @@ module ClassMethods
       #
       # Returns a default policy if no configuration is provided, or a
       # header name and value based on the config.
-      def make_header(config, user_agent = nil)
+      def make_header(config)
```

Comment:
Removing the `user_agent` argument is a breaking change that will require a deprecation release.